tests/dataset/test_dataset.py

```diff
@@ -0,0 +1,330 @@
+"""Test RDF serialisation."""
+
+# pylint: disable=invalid-name,too-many-locals
+
+from pathlib import Path
+
+import pytest
+
+pytest.importorskip("yaml")
+pytest.importorskip("requests")
+
+thisdir = Path(__file__).resolve().parent
+testdir = thisdir.parent
+inputdir = testdir / "input"
+outputdir = testdir / "output"
+
+
+def test_get_context():
+    """Test get_context()."""
+    from tripper.dataset import get_context
+
+    context = get_context()
+    assert isinstance(context, dict)
+    assert "@version" in context
+    assert len(context) > 20
```

Comment:
I do not understand where the context is found. It is not referring to any triplestore as far as I can see?